src/python/grpcio_tests/tests_aio/unit/abort_test.py

```diff
@@ -136,6 +136,7 @@ class TestAbort(AioTestBase):
 
         with self.assertRaises(aio.AioRpcError) as exception_context:
             await call.read()
+            await call.read()
```

Comment:
does it mean that the first `call.read()` has to work, if so should we move the first one out of the `with self.assertRaises(....` context scope?

Reply:
There is a race condition.

If the status arrived before calling the first `call.read()`, even the response message is buffered in Core, it will still raise an `AioRpcError` exception. The result of the data race does change due to implementation details (before this PR, the status always arrive first; after this PR, the message always arrive first).

I hope this test case can be neutral to that data race, and save us some head scratching time debugging it.